### DISCUSSION TITLE
关于数据库配置的疑问

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#210] [02.提问] [answered]
现在lamp-cloud用了nacos里的 mysql.yml，用来做连接mysql的业务的配置，请教几个问题：
1，目前看了下里面的配置没有做多数据源的示例，如果做多数据源应该是支持的吧
2，可不可以新建的业务服务不用nacos里的数据库配置 ，而用本地的？包括其他配置 也都只能用nacos上面的吗？

**Comment by zuihou** [comment, marked as answer]
1. 没有，自己集成
2. bootstamp.yml 里面将 mysql.yml 注释，然后自己写数据源配置

**Reply by zqb666** [reply]
自己写数据源配置上放在本地的 还是用 自动生成代码里生成的放在nacos里的对应业务服务名称的yaml文件

**Reply by zuihou** [reply]
> 自己写数据源配置上放在本地的 还是用 自动生成代码里生成的放在nacos里的对应业务服务名称的yaml文件

随便，只要搞清楚 common.yml、mysql.yml、 lamp-xxx-server.yml 、application.yml 的优先级和加载规则，爱放那放哪
